sdk/monitor/azure-monitor-query/src/main/java/com/azure/monitor/query/models/AggregationType.java

```diff
@@ -53,6 +55,14 @@ public static AggregationType fromString(String value) {
     @JsonValue
     @Override
     public String toString() {
-        return this.value;
+        return this.name;
+    }
+
+    /**
+     * Returns the integer value representing this aggregation.
```

Comment:
Should we add comment on why this value is exposed or mention that this is the service internal representation of aggregation? If I look at just the type it is hard to understand what 'value' represents. 

Also, do we need to expose this to the customer? Is it just public because the other package needs to consume this?